src/components/Checkout/index.js

```diff
@@ -200,6 +207,12 @@ export const Checkout = (props) => {
         setErrors(confirmApplePayError)
       }
     }
+    if (cartResult?.paymethod_data?.gateway === 'credomatic' &&
```

Comment:
Don't hardcode this gateway, create an array and add it, in order to optimize components